.github/workflows/check-release.yml

```diff
@@ -9,10 +9,10 @@ jobs:
   check-version:
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v3
+    - uses: actions/checkout@v4
 
     - name: Set up Python
-      uses: actions/setup-python@v3
+      uses: actions/setup-python@v5
       with:
         python-version: '3.10'
     
```

Comment:
Worth adding `workflow_dispatch:` to this as well incase something fails unexpectedly (as it did for framework) so you don't have to wait till another push on `main` to trigger it!

Reply:
Done.